eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Wallet.hs

```diff
@@ -468,6 +468,8 @@ evaluateTransactionFee ::
   Tx era ->
   -- | The number of key witnesses still to be added to the transaction.
   Word ->
+  -- | The total size of reference scripts
+  Int ->
```

Comment:
Indeed, it's used in two tests: `Fees` in shelley-test and `AlonzoAPI` in cardano-ledger-test.

What do you think about replacing there the usage of `evaluateTransactionFee` with `estimateMinFeeTx`  and passing zero as number of byron witnesses, since the former doesn't set `bootAddrTxWitsL`. 
In this way we'll have more tests for the function that we're exposing, `estimateMinFeeTx` and fewer variations of this function.

Reply:
> What do you think about replacing there the usage of evaluateTransactionFee with estimateMinFeeTx

Yeah, let's do that. `evaluateTransactionFee` is just dumber version of `estimateMinFeeTx`